GET2019-E19-1219/src/Assignment1/ShoppingCart.java

```diff
@@ -0,0 +1,222 @@
+package Assignment1;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class ShoppingCart {
+
+	/**
+	 * AllProductList store product available to purchase
+	 */
+	private ArrayList<Product> AllProductList = new ArrayList<Product>();

```

Comment:
Try to make class generic and use modular approach. Try to keep class specific data like all product list not required here.

Reply:
ok sir in my upcoming assignment i will take modular approach and keep class specific data in there respective class